src/overlays/actors/ovl_Boss_Ganon2/z_boss_ganon2.c

```diff
@@ -278,116 +304,3125 @@ static ColliderJntSphInit D_80907014 = {
+            this->unk_3B0.x = player->actor.world.pos.x;
+            this->unk_3B0.y = player->actor.world.pos.y + 40.0f;
+            this->unk_3B0.z = player->actor.world.pos.z;
+            if (this->unk_398 == 0xA6) {
+                temp_v0_2 = Gameplay_GetCamera(globalCtx, 0);
+                temp_v0_2->eye = this->unk_3A4;
+                temp_v0_2->eyeNext = this->unk_3A4;
+                temp_v0_2->at = this->unk_3B0;
+                func_800C08AC(globalCtx, this->unk_39E, 0);
+                this->unk_39E = 0;
+                func_80064534(globalCtx, &globalCtx->csCtx);
+                func_8002DF54(globalCtx, &this->actor, 7);
+                this->unk_39C = 6;
+            }
+            break;
+        case 6:
+            this->unk_339 = 0x17;
+            temp_f14 = this->unk_1B8.x - player->actor.world.pos.x;
+            temp_f12 = this->unk_1B8.z - player->actor.world.pos.z;
+            temp_a0_2 = Math_Atan2S(temp_f12, temp_f14) - player->actor.shape.rot.y;
+            if ((ABS(temp_a0_2) < 0x2000) && (sqrtf(SQ(temp_f14) + SQ(temp_f12)) < 70.0f) &&
+                (player->swordState != 0) && (player->heldItemActionParam == 3)) {
+                func_80064520(globalCtx, &globalCtx->csCtx);
+                this->unk_39E = Gameplay_CreateSubCamera(globalCtx);
+                Gameplay_ChangeCameraStatus(globalCtx, 0, 1);
+                Gameplay_ChangeCameraStatus(globalCtx, this->unk_39E, 7);
+                this->unk_39C = 7;
+                this->unk_398 = 0;
+                Animation_MorphToPlayOnce(&this->skelAnime, &object_ganon2_Anim_003B1C, 0.0f);
+                this->unk_194 = Animation_GetLastFrame(&object_ganon2_Anim_003B1C);
+                globalCtx->startPlayerCutscene(globalCtx, &this->actor, 0x61);
+            } else {
+                break;
+            }
+        case 7:
+            this->unk_339 = 0x17;
+            Math_ApproachZeroF(&globalCtx->envCtx.unk_D8, 1.0f, 0.2f);
+            player->actor.world.pos.x = 250.0f;
+            player->actor.shape.rot.y = -0x4000;
+            player->actor.world.pos.z = 30.0f;
+            if ((this->unk_398 == 0x14) || (this->unk_398 == 0x1E) || (this->unk_398 == 0x41) ||
+                (this->unk_398 == 0x28)) {
+                func_80078884(NA_SE_VO_LI_SWORD_N);
+                func_80078884(NA_SE_IT_SWORD_SWING_HARD);
+            }
+            if ((this->unk_398 == 0x16) || (this->unk_398 == 0x23) || (this->unk_398 == 0x48) ||
+                (this->unk_398 == 0x2D)) {
+                func_80078884(NA_SE_EN_MGANON_DAMAGE);
+                func_80078884(NA_SE_IT_SHIELD_BOUND);
+                globalCtx->envCtx.unk_D8 = 1.0f;
+            }
+            if ((this->unk_398 == 0x16) || (this->unk_398 == 0x23) || (this->unk_398 == 0x48) ||
+                (this->unk_398 == 0x2D)) {
+                func_8090109C(this, globalCtx);
+            }
+            if ((this->unk_398 >= 0x22U) && (this->unk_398 < 0x28U)) {
```

Comment:
```suggestion
            if ((this->unk_398 >= 0x22) && (this->unk_398 < 0x28)) {
```
I assume they aren't required?